src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -433,7 +434,7 @@ def load_arguments(self, _):
         # managed cluster
         c.argument('ssh_key_value', type=file_type, completer=FilesCompleter(), validator=validate_ssh_key_for_update)
         c.argument('load_balancer_managed_outbound_ipv6_count', type=int)
-        c.argument('outbound_type', arg_type=get_enum_type(outbound_types))
+        c.argument('outbound_type', arg_type=get_enum_type(outbound_types), validator=validate_outbound_type_in_update)
```

Comment:
A simpler way is to declare a new list which only includes lb and managed nat, and set `arg_type=get_enum_type(supported_updated_outbound_types)`. Then you don't need the validator and ut for that.

Reply:
Here a new validator is added to strengthen the business rules（only lb and managed natgateway are allowed) and it is easy to get confused because there are two allowed list..